doc/nrf/libraries/bluetooth_services/rpc.rst

```diff
@@ -81,27 +91,83 @@ Then, you can invoke build command like this:
+
+Additionally, you can use the following options:
+
+  * :kconfig:option:`CONFIG_BT_RPC`
+  * :kconfig:option:`CONFIG_BT_RPC_CLIENT`
+  * :kconfig:option:`CONFIG_BT_RPC_HOST`
+  * :kconfig:option:`CONFIG_BT_RPC_STACK`
+  * :kconfig:option:`CONFIG_BT_RPC_INITIALIZE_NRF_RPC`
+  * :kconfig:option:`CONFIG_BT_RPC_GATT_SRV_MAX`
+  * :kconfig:option:`CONFIG_BT_RPC_GATT_BUFFER_SIZE`
+  * :kconfig:option:`CONFIG_BT_RPC_INTERNAL_FUNCTIONS`
+  * :kconfig:option:`CONFIG_CBKPROXY_OUT_SLOTS`
+  * :kconfig:option:`CONFIG_CBKPROXY_IN_SLOTS`
+
+For more details, see the Kconfig option description.
+
+Samples using the library
+*************************
 
-* :ref:`zephyr:bt_gap`
-* :ref:`zephyr:bluetooth_connection_mgmt`
-* :ref:`zephyr:bt_gatt`
-* :ref:`Bluetooth Cryptography <zephyr:bt_crypto>`
+The following |NCS| sample and application use this library:
 
-The behavior of the implementation is almost the same as Zephyr's with the following exceptions:
+* :ref:`ble_rpc_host`
+* :ref:`ipc_radio`
 
-* The latency is longer because of the overhead for exchanging messages between cores.
-* The :c:func:`bt_gatt_cancel` function is not implemented.
-* The ``flags`` field of  the :c:struct:`bt_gatt_subscribe_params` structure is atomic, so it cannot be correctly handled by the nRF RPC.
-  The library implements the following workaround for it:
+The :ref:`ble_rpc_host` sample exposes the Bluetooth LE stack functionality that runs on an MCU with radio (for example, the nRF5340 network core) to another CPU using the :ref:`nrfxlib:nrf_rpc`.
+When building samples for the application core, enable the :kconfig:option:`CONFIG_BT_RPC_STACK` Kconfig option to run the Bluetooth LE stack on the network core.
+This option builds :ref:`ble_rpc_host` automatically as a child image.
+For more details, see :ref:`ug_nrf5340_building`.
+
+The :ref:`ipc_radio` application is an alternative to the :ref:`ble_rpc_host` sample.
+
+Limitations
+***********
+
+The library currently supports serialization of the following:
+
+  * :ref:`zephyr:bt_gap`
+  * :ref:`zephyr:bluetooth_connection_mgmt`
+  * :ref:`zephyr:bt_gatt`
+  * :ref:`Bluetooth Cryptography <zephyr:bt_crypto>`
+
+The behavior of the Bluetooth implementation is almost the same as Zephyr's with the following exceptions:
+
+  * The latency is longer because of the overhead for exchanging messages between cores.
+    The Bluetooth LE API is not strictly real-time by design, so the additional latency introduced by the IPC communication should be acceptable in most applications.
+    To reduce the latency consider using a different transport backend for nRF RPC, see :ref:nrf_rpc_architecture for details.
+  * Using advanced Bluetooth LE configurations, such as multiple simultaneous connections or advanced security features can be a limitation as the child image (:ref:`ble_rpc_host` or :ref:`ipc_radio`) may require significantly more memory that may not be available on the MCU it runs on.
```

Comment:
```suggestion
  * Using advanced Bluetooth LE configurations, such as multiple simultaneous connections or advanced security features can be a limitation, because the child image (:ref:`ble_rpc_host` or :ref:`ipc_radio`) might require significantly more memory than available on the MCU it runs on.
```